README.md

```diff
@@ -1,13 +1,23 @@
 # SymbolServer
 
+[![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://julia-vscode.github.io/SymbolServer.jl/dev)
```

Comment:
I think the URL will be https://www.julia-vscode.org/SymbolServer.jl/dev, because redirect the whole org to that domain.